modules/storage/config.go

```diff
@@ -30,6 +31,14 @@ func (cfg *Config) RegisterFlagsAndApplyDefaults(prefix string, f *flag.FlagSet)
 	f.Float64Var(&cfg.Trace.WAL.BloomFP, util.PrefixConfig(prefix, "trace.wal.bloom-filter-false-positive"), .05, "Bloom False Positive.")
 	f.IntVar(&cfg.Trace.WAL.IndexDownsample, util.PrefixConfig(prefix, "trace.wal.index-downsample"), 100, "Number of traces per index record.")
 
+	cfg.Trace.Azure = &azure.Config{}
+	f.StringVar(&cfg.Trace.Azure.StorageAccountName, util.PrefixConfig(prefix, "trace.azure.storage_account_name"), "", "azure account name.")
+	f.StringVar(&cfg.Trace.Azure.StorageAccountKey, util.PrefixConfig(prefix, "trace.azure.storage_account_key"), "", "azure access key.")
+	f.StringVar(&cfg.Trace.Azure.ContainerName, util.PrefixConfig(prefix, "trace.azure.container_name"), "", "azure container to store blocks in.")
+	f.StringVar(&cfg.Trace.Azure.Endpoint, util.PrefixConfig(prefix, "trace.azure.endpoint"), "", "azure endpoint to push blocks to.")
+	f.IntVar(&cfg.Trace.Azure.MaxRetries, util.PrefixConfig(prefix, "trace.azure.max_retries"), 3, "Number of retries before give up.")
+	f.BoolVar(&cfg.Trace.Azure.DevelopmentMode, util.PrefixConfig(prefix, "trace.azure.development_mode"), false, "Use azurite as the backend storage.")
```

Comment:
Couple of points
- The `azure` option needs to be documented in the `trace.backend` flag above
- Hyphens(`-`) are preferred in the flags as opposed to underscores (`_`)
- We are trying to keep the number of flags to a minimum and would like to expose only the most commonly used configuration options through them. Some of these can be dropped.. like `max_retries`